code/modules/admin/smites/tabletidestationwide.dm

```diff
@@ -0,0 +1,15 @@
+// Tableslams the target across every table on the station
+/datum/smite/tabletidestationwide
+	name = "TABLETIDE STATION WIDE"
+
+/datum/smite/tabletidestationwide/effect(client/user, mob/living/target)
+	priority_announce("[target] has brought the wrath of the gods upon themselves and is now being tableslammed across the station. Please stand by.")
+	var/list/areas = list()
+	for(var/area/A in world)
+		if(A.z == SSmapping.station_start)
+			areas += A
+	SEND_SOUND(target, sound('sound/misc/slamofthenorthstar.ogg',volume=40))
+	for(var/area/A in areas)
+		for(var/obj/structure/table/TABLE in A)
+			TABLE.tablepush(target, target)
+			sleep(1)
```

Comment:
```suggestion
	for(var/area/area in world)
		if(area.z == SSmapping.station_start)
			areas += area
	SEND_SOUND(target, sound('sound/misc/slamofthenorthstar.ogg',volume=40))
	for(var/area/area in areas)
		for(var/obj/structure/table/TABLE in area)
			TABLE.tablepush(target, target)
			sleep(1)
```
Single letter variables bad.

Reply:
i am the big dum and i am the big sleepy

at least i made table a non single letter *dab